libcalico-go/lib/backend/syncersv1/updateprocessors/felixnodeprocessor.go

```diff
@@ -197,6 +223,22 @@ func (c *FelixNodeUpdateProcessor) Process(kvp *model.KVPair) ([]*model.KVPair,
 			Value:    vxlanTunlMac,
 			Revision: kvp.Revision,
 		},
+		{
+			Key: model.HostConfigKey{
```

Comment:
I thought we said that we could simply pull this off the v3 node, which is being passed directly to Felix already? 

I think we can do without the HostKey types for new fields, because all felixes from the last like 2 years or so have been receiving v3 Node objects in addition to host configs. 

@coutinhop

Reply:
Oh, I'm sorry, I had misunderstood and thought this only meant the IPv6 address... I just merged this PR, I'll make another one to correct this.